designer/client/cypress/e2e/creatorToolbar.cy.ts

```diff
@@ -26,6 +26,7 @@ describe("Creator toolbar", () => {
         cy.contains(/^types$/i).click();
         cy.contains(/^services$/i).click();
         cy.contains(/^sinks$/i).click();
+        cy.contains(/^stickyNotes$/i).click();
```

Comment:
Label looks like technical name of variable. Can you change it to sth like "sticky notes" ?

Reply:
Yeah, it is going to look nicer - changed